js/apps/admin-ui/cypress/e2e/users_test.spec.ts

```diff
@@ -442,4 +443,17 @@ describe("User creation", () => {
 
     listingPage.itemExist(itemIdWithCred, false);
   });
+
+  describe("Accessibility tests for users", () => {
+    beforeEach(() => {
+      loginPage.logIn();
+      keycloakBefore();
+      sidebarPage.goToUsers();
+      cy.injectAxe();
+    });
+
+    it("Check a11y violations on load/ users", () => {
+      cy.checkA11y(undefined, a11YOptions);
```

Comment:
Since these options are the same in every file, perhaps we can [overwrite this command](https://docs.cypress.io/api/cypress-api/custom-commands#Overwrite-Existing-Commands) so it always has these options? This way we can call `cy.checkA11y()` without any arguments, avoiding the need to keep passing in the options.

Reply:
thanks @jonkoops this sounds good to me. I have added a custom command.